ci-operator/config/openshift/release/openshift-release-master__nightly-4.7.yaml

```diff
@@ -321,6 +329,71 @@ tests:
         requests:
           cpu: 100m
     workflow: openshift-e2e-powervs-upi
+- as: e2e-cnv-deploy
+  interval: 12h
+  steps:
+    cluster_profile: azure4
+    test:
+    - as: test
+      cli: latest
+      commands: make deploy_test
+      credentials:
+      - mount_path: /var/run/cnv-ci-brew-pull-secret
+        name: cnv-ci-brew-pull-secret
+        namespace: test-credentials
+      env:
+      - default: '|7820aea2-0d75-11e7-9259-28d244ea5a6d.hhav.f63e13'
+        name: BREW_IMAGE_REGISTRY_USERNAME
+      - default: /var/run/cnv-ci-brew-pull-secret/token
+        name: BREW_IMAGE_REGISTRY_TOKEN_PATH
+      - default: v0.36.0
+        name: KUBEVIRT_RELEASE
+      - default: "4.7"
+        name: OCP_VERSION
+      - default: /tmp/artifacts
+        name: ARTIFACTS_DIR
+      - default: openshift-cnv
+        name: TARGET_NAMESPACE
+      - default: "true"
+        name: PRODUCTION_RELEASE
+      from: cnv-ci-src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 600Mi
+    workflow: ipi-azure
+- as: e2e-cnv-upgrade
```

Comment:
I'd expect this to be called something like `e2e-azure-upgrade-cnv`

Reply:
Done.